mars/oscar/api.py

```diff
@@ -79,7 +79,7 @@ def setup_cluster(address_to_resources: Dict[str, Dict[str, Number]]):
 class Actor(_Actor):
     @classmethod
     def default_uid(cls):
-        return cls.__name__
+        return cls.__name__.encode()
```

Comment:
Why encode the class name into bytes?

Reply:
When actor instance is created in an  actor pool, uid is encoded to bytes and used as the key of the actor instance. Will encoding to bytes make the code more consistent?

Reply:
reverted